google-cloud-storage/src/main/java/com/google/cloud/storage/HttpStorageOptions.java

```diff
@@ -323,6 +323,16 @@ public HttpStorageRpcFactory() {}
     public ServiceRpc create(StorageOptions options) {
       if (options instanceof HttpStorageOptions) {
         HttpStorageOptions httpStorageOptions = (HttpStorageOptions) options;
+        // todo: In the future, this step will be done automatically, and the getResolvedApiaryHost
```

Comment:
Is there a tracking bug to know when this will be addressed?

Reply:
Added!